attractions/time-picker/time-picker.svelte

```diff
@@ -220,7 +226,10 @@
             on:click={() =>
               setHours(
                 hour +
-                  12 * (f12hours && (currentAmPm === 'PM') ^ (value === 12))
+                  12 *
+                    Number(
+                      (f12hours && currentAmPm === 'PM') !== (value === 12)
```

Comment:
In particular, this is one the errors reported by `svelte-check` that makes sense. `value` is of type `Date | null`, which means that the condition `value === 12` will always be `false`

Reply:
Yeah, that's probably a bug)

Reply:
So should I just remove the `!== (value === 12)` part? Also, is there a specific reason it was `^` instead of `!=` previously?

Reply:
You can remove it, but I suggest you don't, since it's kind of out of scope for this PR. Instead we should follow up with a PR that fixes other issues in our code, apart from classes.

Answering the question about `^` – no, I'm pretty sure there was no reason

Reply:
I mean, everything that doesn't involve adding `@type` before `export { _class as class }` is kinda out of scope already)
If you prefer, I can drop the relevant commit from this branch and move it along with the additional fixes in another PR, but they just felt like they belong since it's all about fixing type errors. Also, just removing `!== (value === 12)` doesn't feel like it deserves its own PR)

Reply:
> doesn't feel like it deserves its own PR

I feel like it does. There's a reason why this condition was there and this error might indicate that a bug had crept in our code. We need to consider it separately.

As for moving this commit to another branch – no, I think it's fine here. It's out of scope but it's not that out of scope :)